jobs/email/EditorialReminder.php

```diff
@@ -92,30 +93,31 @@ public function handle(): void
                 $reviewRoundDao = DAORegistry::getDAO('ReviewRoundDAO');
                 $reviewRound = $reviewRoundDao->getLastReviewRoundBySubmissionId($submission->getId(), $submission->getData('stageId'));
 
-                if ($reviewRound->getStatus() === ReviewRound::REVIEW_ROUND_STATUS_PENDING_REVIEWERS) {
+                if ($reviewRound->determineStatus() === ReviewRound::REVIEW_ROUND_STATUS_PENDING_REVIEWERS) {
```

Comment:
using the `determineStatus` seems easier way rather than retrieving the last review assignment and making decisions based on that as the `determineStatus` does that internally .

Reply:
Shouldn't correct status be already determined? If we determine status and it's different from what is stored in the database, it looks wrong to me. What do you think?

Reply:
> If we determine status and it's different from what is stored in the database, it looks wrong to me

 That's 100% right and I am still trying to figure out how such as case, but still unable to . However found one issue , see https://github.com/pkp/pkp-lib/pull/10332/files#r1766427232 . I am running bit more tests to see if I can reproduce a case where `getStatus` and `determineStatus` provide different result .

Reply:
see this https://github.com/pkp/pkp-lib/issues/10214#issuecomment-2372799562